sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeSentimentTests.cs

```diff
@@ -73,19 +73,19 @@ public async Task AnalyzeSentimentWithLanguageTest()
         }
 
         [Test]
-        public async Task AnalyzeSentimentWithOpinionMining()
+        public async Task AnalyzeSentimentWithAssessmentMining()
```

Comment:
because the feature is still called OpinionMining, lets leave the name of the tests with that.
So use `AnalyzeSentimentWithOpinionMining` instead of `AnalyzeSentimentWithAssessmentMining`.

this comment applies to all other test renames